app/renderer/js/utils/sentry-util.ts

```diff
@@ -1,11 +1,12 @@
 import { init } from '@sentry/electron';
 
 import isDev = require('electron-is-dev');
+require('dotenv').config({ path: __dirname + '/../../../../.env' });
```

Comment:
@priyank-p, @vsvipul is this correct in ts?

Reply:
Yes, it is valid though I think linter will complain. Also for consistency let's do:
```suggestion
import dotenv = require('dotenv');
dotenv.config({ path: __dirname + '/../../../../.env' });
```

Reply:
Also, looking at this, let's also import path and pass this path through `path.resolve`.

Reply:
Oh, I was just about to push the required changes.
Thanks @priyank-p!